docs/csharp/codedoc.md

```diff
@@ -12,16 +12,464 @@ ms.devlang: dotnet
 ms.assetid: 8e75e317-4a55-45f2-a866-e76124171838
 ---
 
-# 🔧 Documenting your code
-
-> **Note**
-> 
-> This topic hasn’t been written yet! 
->
-> We welcome your input to help shape the scope and approach. You can track the status and provide input on this
-> [issue](https://github.com/dotnet/core-docs/issues/494) at GitHub.
-> 
-> If you would like to review early drafts and outlines of this topic, please leave a note with your contact information in the issue.
->
-> Learn more about how you can contribute on [GitHub](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md).
->
+# XML Documentation
+
+by [Toni Solarin-Sodara](https://github.com/tsolarin)
+
+XML documentation tags are specially formatted comments, added above the definition of a class 
+or its members, used to document your code. You can generate the XML documentation file at compile time 
+by adding `"xmlDoc": true` under `buildOptions` in your `project.json`
```

Comment:
Instead of "class or its members", I'd use "any user defined type or member". (That's the language from the C# spec). 

(Some tools support namespaces as well, but that's an extension to the standard).